<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ani</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_hab">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>25</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar .hab</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_hab">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>31</y>
      <width>450</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ubicación del archivo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cant_registros">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_base">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar a Base</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>72</y>
      <width>130</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cantidad de registros:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_programa">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>220</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>103</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Programa</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_abrir_directorio">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir ubicación del excel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
